### DISCUSSION TITLE
Tuya web

### DISCUSSION THREAD

**Discussion by devicedev1** [milo526/homebridge-tuya-web#406] [General]
Hello. I use tuya web. When connecting to an account, the plugin does not see devices. Adding by ID does not solve the problem. In the Smart life app, the device is visible and working. Also, this device is not visible in iot.tuya. Were there similar problems?

**Comment by milo526** [comment]
Please post the result of [the steps under "How to check whether the API this library uses can control your device".](https://github.com/milo526/homebridge-tuya-web#how-to-check-whether-the-api-this-library-uses-can-control-your-device)

Without this, it is hard for me to help you with regards to this specific issue.
